webui/react/src/pages/ExperimentDetails.tsx

```diff
@@ -65,6 +46,7 @@ const ExperimentDetails: React.FC = () => {
       ]);
       if (!isEqual(experimentData, experiment)) setExperiment(experimentData);
       if (!isEqual(validationHistory, valHistory)) setValHistory(validationHistory);
+      setIsSingleTrial(experimentData?.config.searcher.name === ExperimentSearcherName.Single);
```

Comment:
chore: Change this to account for experiments with only 1 trial that aren't named single

Reply:
i rebased locally and forgot to force push, so my push failed..